File: projectflutter/lib/data/auth/model/register_request.dart
class RegisterRequest {
  final String firstname;
  final String lastname;
  final String email;
  final String password;
  final DateTime dob;
  final int gender;
  final String phone;

  RegisterRequest(
      {required this.firstname,
      required this.lastname,
      required this.email,
      required this.password,
      required this.dob,
      required this.gender,
      required this.phone});
}

File: Project4/src/main/java/com/example/Project4/dto/auth/RegisterRequest.java
package com.example.Project4.dto.auth;


import java.time.LocalDateTime;

import lombok.*;

@Getter
@Setter
public class RegisterRequest {
    private String firstname;
    private String lastname;
    private String email;
    private String password;
    private String image;
    private LocalDateTime dob;
    private int gender;
    private String phone;
    private String token;
    private String pinCode;
    private boolean status;
    private int roleid;
    private int bmmid;
}
